### ISSUE TITLE
Enable the Docxtemplater Angular expressions parser

### ISSUE THREAD

**Issue by alanorth** [ilri/OpenRXV#49]
Enable the [docxtemplater Angular expressions parser](https://docxtemplater.readthedocs.io/en/latest/angular_parse.html#angular-parser) to allow more complex reporting templates. For example, to use conditional logic in templates:

```
{#users.length>1}
There are multiple users
{/}

{#userName == "John"}
Hello John, welcome back
{/}
```

**Comment by alanorth** [comment]
A user has shared that they would like to be able to create reports where _items are grouped by output type_. See example attached.

[report.pdf](https://github.com/ilri/OpenRXV/files/5580143/report.pdf)

**Comment by ShacMiester** [comment]
@alanorth Can we discuss this on skype?

**Comment by alanorth** [comment]
Yes but I'm out of town until Wednesday afternoon.

**Comment by alanorth** [comment]
@ShacMiester the simple conditional syntax works and we do use it:

```

{#type}
Type: {type}
{/}
```

But conditional expressions like this do not work:

```
{#type == "Journal Article"}
Type: {type}
{/}
```

Even something like this does not work:

```
{#parentcollectionlist.length>1}
parentcollectionlist > 1!!!
{/}
```

So I'm not sure if the extended Angular expressions parser is enabled?

**Comment by ShacMiester** [comment]
Solved! :hammer:

**Comment by alanorth** [comment]
@ShacMiester I cherry-picked the two patches for #49 tested a report with some conditional expressions and I am *still unable to get them to work*.

Did you actually get it working?

**Comment by alanorth** [comment]
Tested on a call with @ShacMiester. Fix was in #74, but I cherry-picked it into master. Closing.
